server/player.lua

```diff
@@ -537,6 +541,7 @@ function CheckPlayerData(source, playerData)
         }
     }
     playerData.gangs = gangs or {}
+    playerData.groups = groups or {}
```

Comment:
What's the purpose of this when we already have gangs and jobs tables?

Reply:
ox_inventory doesn't really differentiate the two, serving them as one combined table simplifies the information transfer to ox_inventory as it also doesn't care about duty status for jobs.

Reply:
Instead of adding to playerData, which is also apart of the public API for Qbox, maybe this should be a derived field computed within ox_inventory qbx bridge layer?